src/main/java/ch/njol/skript/expressions/ExprMessage.java

```diff
@@ -44,103 +45,118 @@
 import ch.njol.util.Kleenean;
 import ch.njol.util.coll.CollectionUtils;
 
-/**
- * @author Peter Güttinger
- */
 @SuppressWarnings("deprecation")
 @Name("Message")
 @Description("The (chat) message of a chat event, the join message of a join event, the quit message of a quit event, or the death message on a death event. This expression is mostly useful for being changed.")
 @Examples({"on chat:",
```

Comment:
Also only 1 tab for the examples as suggested below
```suggestion
@Examples({
	"on chat:",
```